sql/log_event.h

```diff
@@ -4588,6 +4588,8 @@ class Rows_log_event : public Log_event
   uchar    *m_extra_row_data;   /* Pointer to extra row data if any */
                                 /* If non null, first byte is length */
 
+  bool m_unversioned_to_versioned;
```

Comment:
`m_vers_from_nonversioned`

*unversioned* term is wrong, because **un-** means it was versioned once then made non-versioned.